src/main/java/org/eclipse/cargotracker/infrastructure/persistence/DatabaseSetup.java

```diff
@@ -0,0 +1,49 @@
+package org.eclipse.cargotracker.infrastructure.persistence;
```

Comment:
Kindly move this back to the webn.xml. Many of these settings vary from one implementation to the other (e.g. WebLogic) and needs to be updated via Maven variable substitution.